ksqldb-rest-app/src/test/java/io/confluent/ksql/api/AuthTest.java

```diff
@@ -137,6 +142,11 @@ public void close() {
     server.start();
   }
 
+  @Override
+  protected HttpResponse<Buffer> sendGetRequest(final String uri) throws Exception {
+    return sendGetRequestWithCreds(client, uri, USER_WITH_ACCESS, USER_WITH_ACCESS_PWD);
+  }
+
   @Override
   protected HttpResponse<Buffer> sendRequest(final WebClient client, final String uri,
```

Comment:
nit: can we rename this to `sendPostRequest()` in light of the new method above?

Reply:
done